runtime/defaults.vim

```diff
@@ -0,0 +1,72 @@
+" defaults.vim
+"
+" some defaults set early on initialization.
+"
+" based on tpope's sensible.vim[1], with some suggestions gathered 
+" at neovim's issue #276[2]
+"
+" [1]: https://github.com/tpope/vim-sensible
+" [2]: https://github.com/neovim/neovim/issues/276
+
+filetype plugin indent on
+syntax enable
+
+set autoindent
+set backspace=indent,eol,start
+set complete-=i
+set smarttab
+
+set nrformats-=octal
+
+set ttimeout
+set ttimeoutlen=100
```

Comment:
AFAIK we already changed the default of ttimeout[len] (or was it ttimeout[len])?